log4j-core-test/pom.xml

```diff
@@ -371,6 +375,32 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>test-disruptor-4</id>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <additionalClasspathDependencies>
+                <dependency>
+                  <groupId>com.lmax</groupId>
+                  <artifactId>disruptor</artifactId>
+                  <version>${disruptor4.version}</version>
+                </dependency>
+              </additionalClasspathDependencies>
+              <classpathDependencyExcludes>
+                <classpathDependencyExclude>com.lmax:disruptor</classpathDependencyExclude>
+              </classpathDependencyExcludes>
+              <groups>org.apache.logging.log4j.core.test.categories.AsyncLoggers</groups>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Sweet! I hope dependabot can take care of updating `disruptor4.version` in this setting. Otherwise we can duplicate the dependency in the `dependencyManagement` block of _this_ `pom.xml`.

Reply:
Dependabot certainly can not upgrade this: `additionalClasspathDependencies` is new in Surefire 3.2.x.

Also `dependencyManagement` won't help, since we need to compile against version 3.x of Disruptor. Maybe we can put it in `pluginManagement` or a profile that is never used.

Reply:
:+1: A dedicated profile sounds like a good idea.